services/src/main/java/org/apache/unomi/services/services/ProfileServiceImpl.java

```diff
@@ -883,6 +937,7 @@ private void loadPredefinedPropertyTypes(BundleContext bundleContext) {
                 logger.error("Error while loading properties " + predefinedPropertyTypeURL, e);
             }
         }
+        loadPropertyTypesFromPersistence();
```

Comment:
I am not sure about this one, because the new values might still not be available in ES anyway. Is it causing any preactical trouble to initially have empty cache?